hints/is-palindrome.md

```diff
@@ -0,0 +1,31 @@
+<br>
+<details class="hint-accordion">  
+    <summary>Recommended Time & Space Complexity</summary>
+    <p>
+    You should aim for a solution with <code>O(n)</code> time and <code>O(1)</code> space, where <code>n</code> is the length of the input string.
+    </p>
+</details>
+
+<br>
+<details class="hint-accordion">  
+    <summary>Hint 1</summary>
+    <p>
+    A brute force solution would be to create a copy of the string, reverse it, and then check for equality. This would be an <code>O(n)</code> solution with extra space. Can you think of a better way?
```

Comment:
Minor: We can be more specific by saying "Can you think of a way to do this without O(n) space?"

"Better" is a bit ambiguous.

Reply:
A great point to mention for a beginner user. I will be more specific.